scripts/ingests/bdnyc_ingest_photometry.py

```diff
@@ -2,9 +2,80 @@
 
 from astrodbkit2.astrodb import Database, and_
 from sqlalchemy import types  # for BDNYC column overrides
+from datetime import datetime
+from astropy.table import Table
 
 verbose = True
 
+
+# Helper functions
+def fix_band(band):
+    # Fix the band name to what we want in SIMPLE
+    band = band.replace('_', '.')
+    return band
+
+
+def fix_epoch(epoch):
+    # Fix epoch; it's expected to be a decimal year
+
+    def dt_to_dec(dt):
+        """Convert a datetime to decimal year.
+        See: https://stackoverflow.com/questions/29851357/python-datetime-to-decimal-year-one-day-off-where-is-the-bug
+        """
+        year_start = datetime(dt.year, 1, 1)
+        year_end = year_start.replace(year=dt.year + 1)
+        return dt.year + ((dt - year_start).total_seconds() /  # seconds so far
+                          float((year_end - year_start).total_seconds()))  # seconds in year
+
+    decimal_year = None
+    if epoch is not None:
+        try:
+            d = datetime.fromisoformat(epoch)
+            decimal_year = dt_to_dec(d)
+        except:
+            pass
+
+    return decimal_year
+
+
+def get_telescope(band):
+    # Fetch the correct telescope name
+    tel = None
+    if 'WISE' in band:
+        tel = 'WISE'
+    elif 'SDSS' in band:
+        tel = 'SDSS'
```

Comment:
Was this a good assumption? I think I noticed some PanSTARRS photometry also having SDSS bands, but they had correct telescope labels in the BDNYC database. This check would only run if no telescope is provided already.